cloud_library.rst

```diff
@@ -104,6 +103,92 @@ Note that when you push to a library that supports it, {Singularity} 3.7
+.. note::
+
+   The OCI image specification doesn't support SIF signatures, or any additional
+   partitions that can be added to SIF (including OCI-SIF) files by
+   {Singularity}.
+
+   If you have signed an OCI-SIF image locally, the signature(s) will not be
+   pushed to the library. You may wish to push the OCI-SIF, as a single file, to
+   an OCI registry using the ``oras://`` protocol instead.
+
+Pushing OCI-SIF containers to the library in this manner means that they can be
+accessed by other OCI tools. For example, you can use the `Skopeo CLI tool
+<https://github.com/containers/skopeo>`__ to examine the image in the registry
+after it has been pushed. First, ``push`` an OCI-SIF to the SCS ``library://``.
+The ``-U`` option is needed because the image is unsigned.
+
+.. code::
+
+   $ singularity push -U alpine_latest.oci.sif library://example/userdoc/alpine:latest
+   WARNING: Skipping container verification
+   INFO:    Pushing an OCI-SIF to the library OCI registry. Use `--oci` to pull this image.
+
+Now use ``skopeo`` to access the image in the library. This requires
+authentication, which is handled automatically when you use ``singularity
+push``. For other tools, {Singularity} provides a command ``singularity remote
+get-login-password`` that will provide a token that we can use to login to
+``registry.sylabs.io``, which is the address of the OCI registry backing the SCS
+library.
+
+.. code::
+
+   $ singularity remote get-login-password | \
+       skopeo login -u example --password-stdin registry.sylabs.io
+   Login Succeeded!
+
+Finally, use ``skopeo inspect`` to examine the image pushed earlier:
+
+ .. code::
+
+   $ skopeo inspect docker://registry.sylabs.io/example/userdoc/alpine:latest
+   {
+      "Name": "registry.sylabs.io/example/userdoc/alpine",
+      "Digest": "sha256:d08ad9745675812310727c0a99a4472b82fb1cc81e5c42ceda023f1bc35ca34a",
+      "RepoTags": [
+         "latest"
+      ],
+      "Created": "2023-08-07T20:16:26.309461618Z",
+      "DockerVersion": "",
+      "Labels": null,
+      "Architecture": "amd64",
+      "Os": "linux",
+      "Layers": [
+         "sha256:a0c5ced3a57bd1d0d71aaf4a0ea6131d5f163a4a8c5355468c18d4ef006f5d7d"
+      ],
+      "LayersData": [
+         {
+               "MIMEType": "application/vnd.sylabs.image.layer.v1.squashfs",
+               "Digest": "sha256:a0c5ced3a57bd1d0d71aaf4a0ea6131d5f163a4a8c5355468c18d4ef006f5d7d",
+               "Size": 3248128,
+               "Annotations": null
+         }
+      ],
+      "Env": [
+         "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+      ]
+   }
+
+Because the OCI-SIF was pushed as an OCI image, ``skopeo inspect`` is able to
+show the image configuration. This is not possible for non-OCI-SIF images:
+
+.. code::
+
+   $ skopeo inspect docker://registry.sylabs.io/library/default/alpine:latest
+   FATA[0001] unsupported image-specific operation on artifact with type "application/vnd.sylabs.sif.config.v1+json"
```

Comment:
```suggestion
   FATAL[0001] unsupported image-specific operation on artifact with type "application/vnd.sylabs.sif.config.v1+json"
```

Reply:
That's how skopeo error messages are. it's a copy paste. 
<img width="230" alt="image" src="https://github.com/sylabs/singularity-userdocs/assets/4522799/f0e4d296-1f11-43c2-9c86-062e69bed004">